### PR TITLE
add scroll to fixed position API

### PR DESCRIPTION
**Pull request by zilongshanren**
Re: cocos-creator/fireball#3084

Changes proposed in this pull request:
- add scroll to fixed position

@cocos-creator/engine-admins

### CODE REVIEW

File: cocos2d/core/components/CCScrollView.js [hunk 1]
```diff
@@ -484,6 +484,44 @@ var ScrollView = cc.Class({
...
+
+    getScrollOffset: function() {
+        var bottomDeta = Math.abs(this._getContentBottomBoundary() - this._bottomBoundary);
+        var leftDeta = Math.abs(this._getContentLeftBoundary() - this._leftBoundary);
+        var maxScrollOffset = this.getMaxScrollOffset();
+
+        return cc.p(leftDeta, maxScrollOffset.y - bottomDeta);
+    },
+
+    getMaxScrollOffset: function() {
```
Comment (jareguo) [inline cocos2d/core/components/CCScrollView.js:531]: api doc

**Comment by zilongshanren** [comment]
@cocos-creator/engine-admins  examples:  https://github.com/cocos-creator/example-cases/pull/161

File: cocos2d/core/components/CCScrollView.js [hunk 2]
```diff
@@ -484,6 +484,57 @@ var ScrollView = cc.Class({
...
+        var maxScrollOffset = this.getMaxScrollOffset();
+
+        var anchor = cc.pClamp(cc.p(offset.x / maxScrollOffset.x, (maxScrollOffset.y - offset.y ) / maxScrollOffset.y),
+                           cc.p(0, 0), cc.p(1, 1));
+
+        this.scrollTo(anchor, timeInSecond, attenuated);
+    },
+
+    /**
+     * !#en Get the current scroll offset related to scrollview 's top left origin.
```
Comment (jareguo) [inline cocos2d/core/components/CCScrollView.js]: 这样会不会好点 Get the positive offset value value corresponds to the content being offset to the top left.

File: cocos2d/core/components/CCScrollView.js [hunk 3]
```diff
@@ -484,6 +484,57 @@ var ScrollView = cc.Class({
...
+        var maxScrollOffset = this.getMaxScrollOffset();
+
+        return cc.p(leftDeta, maxScrollOffset.y - bottomDeta);
+    },
+
+    /**
+     * !#en Get the maximize available  scroll offset
+     * !#zh 获取滚动视图最大可以滚动的偏移量
+     * @method getMaxScrollOffset
+     * @return {Ve2c} - A Vec2 value indicate the maximize scroll offset in x and y axis.
```
Comment (jareguo) [inline cocos2d/core/components/CCScrollView.js]: Ve2c -> Vec2

File: cocos2d/core/components/CCScrollView.js (hunk 2 above)
Comment (zilongshanren) [inline cocos2d/core/components/CCScrollView.js]: 我改成这个了 Get the positive offset value corresponds to the content's top left boundary.

**Comment by zilongshanren** [comment]
@jareguo  已更新

**Pull request merged**